plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -258,6 +261,7 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
 
    cli.add_options()
          ("genesis-json", bpo::value<bfs::path>(), "File to read Genesis State from")
+         ("chain-id", bpo::value<string>(), "chain_id for the blockchain")
```

Comment:
Ok, I had thought we would want to allow the user to pass a chain-id to validate the snapshot, similar to how it can already pass a genesis-json now, since it will only be able to validate that genesis-state against the chain-id.